src/custom/utils/operator.ts

```diff
@@ -175,5 +193,24 @@ export async function getFeeQuote(chainId: ChainId, tokenAddress: string): Promi
   }
 }
 
+export async function getOrder(chainId: ChainId, orderId: string): Promise<OrderMetaData | null> {
+  console.log('[util:operator] Get order for ', chainId, orderId)
+
+  let response: Response | undefined
+  try {
+    const responseMaybeOk = await _get(chainId, `/orders/${orderId}`)
+    response = responseMaybeOk.ok ? responseMaybeOk : undefined
+  } catch (error) {
+    // do nothing
```

Comment:
Why??

Reply:
we should review this in `getFee` i following that approach and also asking myself the same question. IMO we shouldn't throw in the catch and/or can continue dealing with the return value as we do with `response.ok` @alfetopito @anxolin

Reply:
Let's continue in: https://github.com/gnosis/gp-swap-ui/issues/202